### DISCUSSION TITLE
chia plots check report wrong?

### DISCUSSION THREAD

**Discussion by jooooooooooe** [Chia-Network/chia-blockchain#8277] [Support]
Today, after the update to chia 1.2.5 in did a chia plots check -n 5 which reported 5 invalid plots. Then I did a second run with -n 50. This reported 7 invalid plots, but totally different plots than before.... Is this a bug or is this intended? Shall I delete the invalid ones?

**Comment by loppefaaret** [comment]
uhh, that does not sound good, but before you start deleting plot files - getting these random results could point to something else but the plot themselfs being the issue!
running plots check should send the same static challenges to the files - so the `-n 50` should run the same 5 checks first, as when you did `-n 5` - meaning those same files should report the same errors (+ potential more)

**Reply by jooooooooooe** [reply]
Woooo. Thats what I thought. But: what is wrong? I did not notice this until now. I don't know, why I ran the exact test twice. I guess, what puzzled me, was that quite old plots were suddenly invalid. Should I downgrade to 1.2.4 or just sit and wait for 1.2.6 ;-)?

**Reply by loppefaaret** [reply]
i would try and test hardware...

**Reply by jooooooooooe** [reply]
Hmmm. I think I leave it as it is and wait for the next update. Since my Farm runs on a macmini 2014 where almost everything is soldered on and everything else runs great, I don't want to mess with the system. Maybe on next reboot I'll do the Apple Hardware Test...

**Comment by BrandtH22** [comment]
@jooooooooooe , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
